src/pages/index.js

```diff
@@ -123,122 +112,62 @@ export default function IndexPage({ pkgVersion }) {
-              {' '}
-              ♥️{' '}
-            </span>
-            by the team at LifeOmic.
-          </p>
-          <dl className="flex flex-wrap justify-center whitespace-no-wrap text-xs font-bold leading-5 sm:text-sm">
-            <div className="flex items-center mx-3 space-x-2 text-orange-300 sm:mx-4 xl:ml-0 xl:mr-8">
-              <dt>
-                <Lifeology className="h-6 w-6" role="img" aria-hidden />
-              </dt>
-              <dd className="uppercase">
-                {Object.keys(allLinedChromicons).length} icons
-              </dd>
-            </div>
-            <div className="flex items-center mx-3 space-x-2 text-teal-300 sm:mx-4 xl:ml-0 xl:mr-8">
-              <dt>
-                <CheckCircle className="h-6 w-6" role="img" aria-hidden />
-              </dt>
-              <dd className="uppercase">MIT Licensed</dd>
-            </div>
-            <div className="flex items-center mx-3 space-x-2 text-purple-400 sm:mx-4 xl:ml-0 xl:mr-8">
-              <dt>
-                <Flag className="h-6 w-6" role="img" aria-hidden />
-              </dt>
-              <dd className="uppercase">Version {pkgVersion}</dd>
-            </div>
-          </dl>
+          <div>
+            <img
+              src={`${process.env.BACKEND_URL}/big-icon.png`}
+              className="absolute"
+              alt="home banner"
+              style={{
+                bottom: -16,
+                right: -5,
+                top: -75,
+                zIndex: -1,
+              }}
+            />
+          </div>
         </div>
-        <SearchField
-          className="mt-10 w-full sm:px-6 md:px-0 md:mb-0"
-          inputClassName="w-full"
-          value={searchText}
-          onChange={(e) => {
-            const search = e.target.value;
-
-            setSearchText(e.target.value);
-
-            const filteredIcons =
-              selectedTab !== 'all'
-                ? getChromicons()?.filter((icon) =>
-                    icon?.categories?.includes(selectedTab)
-                  )
-                : getChromicons();
-
-            if (!search) {
-              setVisibleIcons(filteredIcons);
-              return;
-            }
-
-            setVisibleIcons(
-              filteredIcons?.filter(
-                (icon) =>
-                  icon.rawName?.toLowerCase().includes(search.toLowerCase()) ||
-                  icon.name?.toLowerCase().includes(search.toLowerCase()) ||
-                  icon?.keywords?.toLowerCase()?.includes(search.toLowerCase())
-              )
-            );
-          }}
-        />
-      </header>
-
-      <main className="bg-white text-gray-600 flex flex-1 flex-col scrolling-touch">
-        <div className="flex justify-between items-center shadow-banner px-4 flex-col sm:px-6 lg:px-16">
+        <div className="lg:flex md:flex sm:flex justify-between items-start shadow-banner px-4 lg:flex-row sm:flex-col sm:px-6 lg:px-6 bg-white">
```

Comment:
Looks like `flex` is wanted at each breakpoint here, any reason not to have `flex` here instead of `lg:flex md:flex sm:flex`?